activesupport/test/cache/stores/mem_cache_store_test.rb

```diff
@@ -45,7 +45,6 @@ def setup
     @namespace = "test-#{SecureRandom.hex}"
     @cache = lookup_store(expires_in: 60)
     @peek = lookup_store
-    @data = @cache.instance_variable_get(:@data)
```

Comment:
I couldn't find anywhere this was used in this file or any of the included modules, and this file's tests continued to pass for me locally. 🤷